app/views/jobs/_system_status.html.erb

```diff
@@ -1,3 +1,20 @@
-
-cluster is: <%= cluster %>
-job size is: <%= jobs.nil? ? 'nil' : jobs.size %>
\ No newline at end of file
+<div class='container border py-1 rounded-1'>
+    <strong>Cluster Name:</strong> <%= cluster %><br>
```

Comment:
```suggestion
    <h2><strong>Cluster Name:</strong> <%= cluster %><h2>
```
This should be an `h2` for accessibility reasons, however it's styled - it makes navigation easier.